.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,53 @@
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#top-level-syntax
+name: Bug Report
+description: File a bug
+title: 'bug: '
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Thanks for taking the time to file a bug!
+
+  - type: textarea
+    attributes:
+      label: Overview
+      description: |
+        Use this section to describe the bug at a high level.
+        Please include any issues you can find that may be related.
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Plugin
+      description: Which package is this about?
+      multiple: false
+      options:
+        - general
+  - type: textarea
+    attributes:
+      label: Environment
+      description: What environment (Python version, OS, hatchling version, etc) are you running the plugin on?
```

Comment:
missing validation? (and I think this should be required)

Reply:
But it wouldn't if this was an issue about something repo-wide